server/neptune/workflows/internal/root/root.go

```diff
@@ -12,8 +12,8 @@ type Root struct {
 	// Path is the relative path from the repo
 	Path      string
 	TfVersion string
-	Apply     job.Job
```

Comment:
thoughts on wrapping this in a `job.Apply` struct for readability?

Reply:
not sure how that really helps with readability?

Reply:
maybe keep `job.Terraform` for the apply job/general job and rename the plan specific struct to`job.TerraformPlan` (assuming you mean trying to maintain the connection between the two job structs)? either way I think it's okay